docs/source/documentation/api_tokens_generation.rst

```diff
@@ -3,7 +3,24 @@ How-to use new API tokens for login
 
 This short guide explains how to manage API tokens in the web application, and use them in the Substra SDK.
 
-.. note::
+.. admonition:: Why generate API tokens?
+
+   The Substra SDK provides a way to log in using username and password (see `substra.Client <references/sdk.html#client>`_).
+   
+   It is safe, but should be used with caution:
+   
+   * It doesn't allow for a precise lifetime or separating concerns by creating one token per purpose.
+   
+   * It may surprise or limit you through its underlying automated session management.
+   
+   * It can encourage using cleartext passwords, which can end up shared in version control.
+   
+   For these reasons, it is possible for Substra node administrators to disable "implicit login" and force users to generate tokens in the web app.
```

Comment:
Oh I didn't know we could do that, is a value to be changed in an helm chart? Is it explained somewhere?

Reply:
It's a new option called `server.allowImplicitLogin` added by https://github.com/Substra/substra-backend/pull/698. We had a discussion on whether it should be on or off by default.

Reply:
Ok thanks. 

You could link to that https://github.com/Substra/substra-backend/blob/4f0ed6dbcb71be34ae54b99db3cd89d1153a2549/charts/substra-backend/README.md#server-settings
once it's merged?